spec/loophp/collection/CollectionSpec.php

```diff
@@ -3999,9 +3999,13 @@ public function it_can_when(): void
 
     public function it_can_window(): void
     {
-        $this::fromIterable(range('a', 'z'))
+        $this::fromIterable(range('a', 'c'))
             ->window(0)
-            ->shouldIterateAs(range('a', 'z'));
+            ->shouldIterateAs([
+                ['a'],
+                ['b'],
+                ['c'],
+            ]);
```

Comment:
is this change a BC-break? Before `window(0)` would return the items essentially unchanged, but now it puts every element in an array

Reply:
Right, this is indeed a minor BC Break. We can even pass negative numbers and get a different behavior that might be useful. I completely forgot to document that. Thanks.